planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
@MicheleGuerriero, I think #252 does not affect `MonitorDamGenerator`.
Please, take a look [here](https://github.com/kiuby88/SeaCloudsPlatform/commit/504deab1a648be90e58672a2e4c1e60514875f06). I have added to DamGenerator a fist current policy integration. I hope it could be useful to clarify some points.

This code it will not push to SeaClouds (at least we need a minor refactoring before) and I am solving Andrea comments about the policy. But it is only a first example integration.